config.json

```diff
@@ -11,404 +11,404 @@
       "topics": [
       ]
     },
     {
-      "slug": "wordy",
-      "difficulty": 1,
+      "slug": "ocr-numbers",
+      "difficulty": 5,
       "topics": [
       ]
     },
     {
       "slug": "food-chain",
-      "difficulty": 1,
+      "difficulty": 5,
       "topics": [
       ]
     },
     {
-      "slug": "linked-list",
-      "difficulty": 1,
+      "slug": "custom-set",
+      "difficulty": 5,
       "topics": [
       ]
     },
     {
-      "slug": "custom-set",
-      "difficulty": 1,
+      "slug": "simple-cipher",
+      "difficulty": 5,
       "topics": [
       ]
     },
     {
-      "slug": "nth-prime",
-      "difficulty": 1,
+      "slug": "parallel-letter-frequency",
+      "difficulty": 5,
       "topics": [
       ]
     },
     {
       "slug": "palindrome-products",
-      "difficulty": 1,
+      "difficulty": 6,
       "topics": [
       ]
     },
     {
-      "slug": "pascals-triangle",
-      "difficulty": 1,
+      "slug": "pig-latin",
+      "difficulty": 6,
       "topics": [
       ]
     },
     {
-      "slug": "pig-latin",
-      "difficulty": 1,
+      "slug": "robot-name",
+      "difficulty": 6,
       "topics": [
       ]
     },
     {
-      "slug": "pythagorean-triplet",
-      "difficulty": 1,
+      "slug": "bank-account",
+      "difficulty": 6,
       "topics": [
       ]
     },
     {
-      "slug": "saddle-points",
-      "difficulty": 1,
+      "slug": "queen-attack",
+      "difficulty": 6,
       "topics": [
       ]
     },
     {
-      "slug": "say",
-      "difficulty": 1,
+      "slug": "sublist",
+      "difficulty": 6,
       "topics": [
       ]
     },
     {
-      "slug": "secret-handshake",
-      "difficulty": 1,
+      "slug": "linked-list",
+      "difficulty": 6,
       "topics": [
       ]
     },
     {
-      "slug": "series",
-      "difficulty": 1,
+      "slug": "zebra-puzzle",
+      "difficulty": 7,
       "topics": [
       ]
     },
     {
-      "slug": "sieve",
-      "difficulty": 1,
+      "slug": "minesweeper",
+      "difficulty": 7,
       "topics": [
       ]
     },
     {
-      "slug": "simple-cipher",
-      "difficulty": 1,
+      "slug": "wordy",
+      "difficulty": 7,
       "topics": [
       ]
     },
     {
       "slug": "change",
-      "difficulty": 1,
+      "difficulty": 7,
       "topics": [
       ]
     },
     {
-      "slug": "connect",
-      "difficulty": 1,
+      "slug": "say",
+      "difficulty": 8,
       "topics": [
       ]
     },
     {
-      "slug": "parallel-letter-frequency",
-      "difficulty": 1,
+      "slug": "connect",
+      "difficulty": 8,
       "topics": [
       ]
     },
     {
       "slug": "sgf-parsing",
-      "difficulty": 1,
+      "difficulty": 9,
       "topics": [
       ]
     },
     {
       "slug": "go-counting",
-      "difficulty": 1,
+      "difficulty": 9,
       "topics": [
       ]
     },
     {
-      "slug": "zipper",
-      "difficulty": 1,
+      "slug": "lens-person",
```

Comment:
lens-person 9 or 10? If you really want to understand the types, they can take a while to wrap your head around. But once you understand them the rest of the problem is not so hard.